src/main/java/com/minecolonies/colony/Colony.java

```diff
@@ -205,6 +228,18 @@ protected void writeToNBT(NBTTagCompound compound)
         }
         compound.setTag(TAG_CITIZENS, citizenTagList);
 
+        //  Achievements
+        final NBTTagList achievementsTagList = new NBTTagList();
+        for (final Achievement achievement : this.colonyAchievements)
+        {
+            // TODO: Validate this
+            String achievementKey = achievement.statId;
+            NBTTagCompound achievementCompound = new NBTTagCompound();
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Local variable 'achievementCompound' could be declared final [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=pmd%3ALocalVariableCouldBeFinal)